### ISSUE TITLE
VRF Lite Support in Network Services

### ISSUE THREAD

**Issue by jrecchia1029** [aristanetworks/ansible-avd#4785]
### Enhancement summary

Need the ability to configure a VRF in BGP from the Network Services data model that doesn't require one of the existing address families (`evpn`, `vpn-ipv4`, or `vpn-ipv6`) to be enabled.

Currently, the only way to generate BGP config for the VRF is to have `overlay_routing_protocol` for the device set to `ebgp` and the VRF's `address_families` value set to `["evpn"]`.

This produces something like below which isn't exactly correct because the route-target statements are not desired.
```
router bgp 65000
   vrf PROD
      rd 172.16.0.1:10
      route-target import evpn 10:10
      route-target export evpn 10:10
      router-id 172.16.0.1
      update wait-install
      neighbor 172.16.20.1 peer group FW
      neighbor 192.168.255.1 peer group MLAG-IPv4-UNDERLAY-PEER
      redistribute connected
```

### Which component of AVD is impacted

eos_designs

### Use case example

This is necessary for non-VXLAN deployment that leverage VRF Lite.

For example, if I have a non-default VRF in a non-VXLAN environment where the routing protocol being used is BGP, I should be able to generate a VRF in the BGP configuration as seen below.

```
router bgp 65000
   vrf PROD
      router-id 172.16.0.1
      update wait-install
      neighbor 172.16.20.1 peer group FW
      neighbor 192.168.255.1 peer group MLAG-IPv4-UNDERLAY-PEER
      redistribute connected
```

A `route-distinguisher` and `route-target` should be _optional_ to add into the VRF as well.

### Describe the solution you would like

Maybe an extra address family for the VRF called `ipv4` which will allow the VRF configuration to be generated without a `route-distinguisher` or `route-target`

### Contributing Guide

- [X] I agree to follow this project's Code of Conduct

**Comment by gmuloc** [comment]
@jrecchia1029 this should be addressed already in AVD 5.0.0 please see the porting guide: https://avd.arista.com/5.1/docs/porting-guides/5.x.x.html#bgp-is-now-configured-for-network-services-vrfs-even-if-evpn-is-not-part-of-the-address-families

**Comment by jrecchia1029** [comment]
@gmuloc is correct.

Using `bgp.enabled: true` under the VRF solves the issue.
